client/package.json

```diff
@@ -24,18 +24,24 @@
     "@chakra-ui/skip-nav": "2.0.13",
     "@emotion/react": "11.10.5",
     "@emotion/styled": "11.10.5",
+    "@hookform/resolvers": "^2.9.10",
     "chakra-confirm": "1.2.17",
     "chakra-data-table": "1.4.1",
     "chakra-next-link": "1.0.10",
+    "class-transformer": "^0.5.1",
```

Comment:
So it is, thanks for sharing the example. It's a bit of a strange one, since it's a peer dependency of a subpackage and so does not get installed by default.

Anyways, this all looks fine now.